pages/account/concepts.mdx

```diff
@@ -52,12 +52,20 @@ A password is a string of characters associated to your account's email address
 
 ## Single Sign-on (SSO)
 
-Single Sign-On (SSO) allows you to use your Google or Microsoft account to log in to the console. To do so, make sure the email address associated with your Scaleway account matches the email address of your Google or Microsoft account.
+Single Sign-On (SSO) allows you to use your Google or Microsoft account to log in to the console. To do so, make sure the email address associated with your Scaleway account matches the email address of your Google, Microsoft or Github account.
```

Comment:
We need to also add "Github" in the first sentence

`Single Sign-On (SSO) allows you to use your Google, Microsoft or Github account to log in to the console.`

Reply:
```suggestion
Single Sign-On (SSO) allows you to use your Google, Microsoft, or Github account to log in to the console. To do so, make sure the email address associated with your Scaleway account matches the email address of your Google, Microsoft, or Github account.
```